Plugins/NativeCamera/Source/Apple/CameraDevice.mm

```diff
@@ -15,16 +16,17 @@
 #include <Foundation/Foundation.h>
 #include <AVFoundation/AVFoundation.h>
 
-@class CameraTextureDelegate;
-
 #include "../CameraDevice.h"
-#include <napi/napi.h>
+
+@class CameraTextureDelegate;
 
 @interface CameraTextureDelegate : NSObject <AVCaptureVideoDataOutputSampleBufferDelegate>
 {
     @public AVCaptureVideoOrientation VideoOrientation;
 
     CVMetalTextureCacheRef textureCache;
+    CVMetalTextureRef cameraTextureY;
+    CVMetalTextureRef cameraTextureCbCr;
```

Comment:
This is unrelated to ImageCapture, I just noticed this while making other changes. These two members were declared in the @implementation instead of the @interface, which in the bizarre world of Objective C made them effectively static members instead of instance members, which is definitely not the behavior we want.